.github/workflows/gcp-deploy.yaml

```diff
@@ -48,158 +70,192 @@ jobs:
+
+      - name: And That Result Again In Another Step
         id: third
         run: |
-          echo "::set-output name=GRAPHQL_URL::https://${{steps.second.outputs.HASURA_HOST}}/v1/graphql"
-          echo "::set-output name=BACKEND_TAG::${{steps.second.outputs.DOCKER_PATH}}/backend:pr-${{github.event.number}}"
-          echo "::set-output name=HASURA_TAG::${{steps.second.outputs.DOCKER_PATH}}/hasura:pr-${{github.event.number}}"
-          echo "::set-output name=FRONTEND_TAG::${{steps.second.outputs.DOCKER_PATH}}/frontend:pr-${{github.event.number}}"
-          echo "::set-output name=DB_PASSWORD::$(cat /dev/urandom | tr -cd [:alnum:] | head -c 16)"
+          echo "::set-output name=GRAPHQL_URL::\
+          https://${{steps.second.outputs.HASURA_HOST}}/v1/graphql"
+          echo "::set-output name=BACKEND_TAG::\
+          ${{steps.second.outputs.DOCKER_PATH}}/backend:pr-${{github.event.number}}"
+          echo "::set-output name=HASURA_TAG::\
+          ${{steps.second.outputs.DOCKER_PATH}}/hasura:pr-${{github.event.number}}"
+          echo "::set-output name=FRONTEND_TAG::\
+          ${{steps.second.outputs.DOCKER_PATH}}/frontend:pr-${{github.event.number}}"
 
   delete-db:
-    name: Delete Existing Database
+    name: Delete Hasura’s User & Database
     runs-on: ubuntu-latest
     needs: [env]
 
     steps:
-      - name: Set Up gcloud CLI
+      - name: Set Up GCloud CLI
         uses: google-github-actions/setup-gcloud@v0.2.1
         with:
           project_id: ${{needs.env.outputs.PROJECT_ID}}
           service_account_key: ${{secrets.GCP_SA_KEY}}
           export_default_credentials: true
 
-      - name: Delete Hasura’s Database
+      - name: "Delete GCloud SQL User: ${{needs.env.outputs.DB_NAME}}"
+        continue-on-error: true
+        run: |
+          gcloud -q sql users delete ${{needs.env.outputs.DB_NAME}} \
+            -i ${{needs.env.outputs.CLOUDSQL_INSTANCE_NAME}}
+
+      - name: "Delete Database: ${{needs.env.outputs.DB_NAME}}"
         continue-on-error: true
         run: |
           wget -q https://dl.google.com/cloudsql/cloud_sql_proxy.linux.amd64 -O cloud_sql_proxy
           chmod u+x cloud_sql_proxy
-          ./cloud_sql_proxy -instances ${{needs.env.outputs.CLOUDSQL_CONNECTION_NAME}} -dir /tmp/cloudsql &
+          ./cloud_sql_proxy -instances ${{needs.env.outputs.CLOUDSQL_CONNECTION_NAME}} \
+            -dir /tmp/cloudsql &
           PID=$!
           sleep 10
-          PGPASSWORD=${{secrets.GCP_POSTGRES_PASSWORD}} dropdb -h /tmp/cloudsql/${{needs.env.outputs.CLOUDSQL_CONNECTION_NAME}} -U postgres ${{needs.env.outputs.DB_NAME}} -f
+          PGPASSWORD=${{secrets.GCP_POSTGRES_PASSWORD}} \
+            dropdb -h /tmp/cloudsql/${{needs.env.outputs.CLOUDSQL_CONNECTION_NAME}} \
+            -U postgres ${{needs.env.outputs.DB_NAME}} -f
           kill $PID
 
   create-db:
-    name: Create New Database
+    name: "Create New SQL User & Database"
```

Comment:
why does this have quotation marks but the delete step doesnt? also the dleete step mentions hasura but this doesnt (fine with either, but both should be the same imo)

Reply:
The ampersand requires quotes.

Reply:
the delete-db job also got an ampersand but no quotes though. guess you will have to add quotes there then

Reply:
I would've sworn that VSCode told me I needed quotes, but apparently not…